app/models/pglogical_subscription.rb

```diff
@@ -14,12 +14,21 @@ class PglogicalSubscription < ActsAsArModel
     :provider_region_name => :string
   )
 
-  def self.find(*args)
-    subs = pglogical.enabled? ? pglogical.subscriptions : []
-    case args.first
-    when :all then subs.collect { |s| new(subscription_to_columns(s)) }
-    when :first, :last then subs.presence && new(subscription_to_columns(subs.send(args.first)))
-    end
+  delegate :all, :first, :last, :to => :subscriptions
```

Comment:
What does the delegate line here do, if you are overriding the methods immediately afterwards?